core/src/main/java/org/infinispan/topology/LocalTopologyManagerImpl.java

```diff
@@ -103,8 +103,6 @@ public class LocalTopologyManagerImpl implements LocalTopologyManager, GlobalSta
    @Inject PersistentUUIDManager persistentUUIDManager;
    @Inject EventLogManager eventLogManager;
    @Inject CacheManagerNotifier cacheManagerNotifier;
-   // Not used directly, but we have to start the ClusterTopologyManager before sending the join request
-   @Inject ClusterTopologyManager clusterTopologyManager;
```

Comment:
Note the comment we had here, and correct me. Both the local and cluster components are global. The join request comes from the `StateTransferManagerImpl`, a cache component.

I am assuming that any cache component would only start after the global components have started. So, we should always have the ClusterTopologyManager already started when running a join command.